recipes/varint/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "varint" %}
+{% set version = "1.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a6ecc02377ac5ee9d65a6a8ad45c9ff1dac8ccee19400a5950fb51d594214ca5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - varint
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/fmoo/python-varint
+  summary: Simple python varint implementation
+  license: MIT
+  license_file: LICENSE
```

Comment:
Could you add a comment in the above line with the following link to a PR to resolve this issue?

https://github.com/fmoo/python-varint/pull/8

You could also add a link to your issue raising adding the license files (https://github.com/fmoo/python-varint/pull/7). Thanks for doing that.